charts/weave-scope/charts/weave-scope-frontend/templates/_helpers.tpl

```diff
@@ -6,6 +6,53 @@ release: {{ .Release.Name }}
 app: {{ template "toplevel.name" . }}
 {{- end }}
 
+*/ -}}
+{{- define "chartmuseum.name"}}
+{{- $global := default (dict) .Values.global -}}
+{{- $base := default .Chart.Name .Values.nameOverride -}}
+{{- $gpre := default "" $global.namePrefix -}}
+{{- $pre := default "" .Values.namePrefix -}}
+{{- $suf := default "" .Values.nameSuffix -}}
+{{- $gsuf := default "" $global.nameSuffix -}}
+{{- $name := print $gpre $pre $base $suf $gsuf -}}
+{{- $name | lower | trunc 54 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- /*
+
+*/ -}}
+{{- define "chartmuseum.fullname"}}
+{{- $global := default (dict) .Values.global -}}
+{{- $base := default (printf "%s-%s" .Release.Name .Chart.Name) .Values.fullnameOverride -}}
+{{- $gpre := default "" $global.fullnamePrefix -}}
+{{- $pre := default "" .Values.fullnamePrefix -}}
+{{- $suf := default "" .Values.fullnameSuffix -}}
+{{- $gsuf := default "" $global.fullnameSuffix -}}
+{{- $name := print $gpre $pre $base $suf $gsuf -}}
+{{- $name | lower | trunc 54 | trimSuffix "-" -}}
+{{- end -}}
+
+
+{{- /*
+
+*/ -}}
+{{- define "chartmuseum.labels.standard" -}}
+app: {{ template "chartmuseum.name" . }}
+chart: {{ template "chartmuseum.chartref" . }}
+heritage: {{ .Release.Service | quote }}
+release: {{ .Release.Name | quote }}
+{{- end -}}
+
+{{- /*
+
```

Comment:
why we have empty comment block here, can we have some description or remove it?